communitter-frontend/src/pages/communitiesPage.js

```diff
@@ -0,0 +1,5 @@
+import Communities from "../components/Communities";
+
+export function CommunitiesPage() {
+  return <Communities />;
```

Comment:
We can see the private communities in the list, can we remove private communities